### PR TITLE
COMMERCE-8294

### PR DESCRIPTION
**Pull request by brianikim**
@riccardo-alberti 

https://issues.liferay.com/browse/COMMERCE-8294

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceAddressServiceImpl.java [hunk 1]
```diff
@@ -394,7 +395,9 @@ protected void checkPermission(String className, long classPK)
 		if (className.equals(CommerceOrder.class.getName())) {
 			commerceOrderService.getCommerceOrder(classPK);
 		}
-		else if (className.equals(CommerceAccount.class.getName())) {
+		else if (className.equals(CommerceAccount.class.getName()) ||
+				 className.equals(AccountEntry.class.getName())) {
```
Comment (drewbrokke) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceAddressServiceImpl.java:399]: Brian will probably ask for this to be alphabetized.

**Comment by riccardo-alberti** [comment]
@drewbrokke thank you

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
@brianikim could you please rebase? this one is ready to go. thx

**Comment by brianchandotcom** [comment]
@brianikim @riccardo-alberti I merged this in manually.

**Pull request closed without merging**